docs/releases.md

```diff
@@ -5,9 +5,46 @@ title: Releases
 
 ## What's new in Graphback
 
+
 This file contains changes and migration steps for Graphback project. 
 Please follow indidividual releases for more information.
 
+
+# 0.10.0
```

Comment:
@craicoverflow I would love to get your updates for that as well. What we have changed since 0.9.0 :D